sdk/python/foundation-models/system/finetune/video-multi-object-tracking/mmtracking-video-multi-object-tracking.ipynb

```diff
@@ -0,0 +1,1177 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from PIL import Image\n",
+    "sample_image = os.path.join(dataset_dir, \"train/MOT17-02-FRCNN/img1/000001.jpg\")\n",
+    "sample_image = Image.open(sample_image)\n",
+    "print(sample_image.size)\n",
+    "sample_image"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.2 Upload the images to Datastore through an AML Data asset (URI Folder)\n",
+    "\n",
+    "In order to use the data for training in Azure ML, we upload it to our default Azure Blob Storage of our  Azure ML Workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uploading image files by creating a 'data asset URI FOLDER':\n",
+    "\n",
+    "from azure.ai.ml.entities import Data\n",
+    "from azure.ai.ml import Input\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "\n",
+    "my_data = Data(\n",
+    "    path=dataset_dir,\n",
+    "    type=AssetTypes.URI_FOLDER,\n",
+    "    description=f\"{dataset_name} dataset folder\",\n",
+    "    name=f\"{dataset_name}_sample_folder\",\n",
+    ")\n",
+    "\n",
+    "uri_folder_data_asset = workspace_ml_client.data.create_or_update(my_data)\n",
+    "\n",
+    "# or if the uri_folder was uploaded, we could get it with:\n",
+    "# uri_folder_data_asset = workspace_ml_client.data.get(name = f\"{dataset_name}_sample_folder\", version=1)\n",
+    "\n",
+    "print(uri_folder_data_asset)\n",
+    "print(\"\")\n",
+    "print(\"Path to folder in Blob Storage:\")\n",
+    "print(uri_folder_data_asset.path)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.3 Convert the downloaded data to JSONL\n",
+    "\n",
+    "In this example, the MOT17_tiny dataset is stored in a directory. There are two different folders inside `train` image folder, each denoting a different video:\n",
+    "\n",
+    "- train/MOT17-02-FRCNN\n",
+    "- train/MOT17-04-FRCNN\n",
+    "\n",
+    "This is the most common data format for image object tracking. Inside each of video folder, video frames are sorted in sequence.\n",
+    "\n",
+    "Note that, in the above folders, videos are already parsed into image frames. If you have a video available at hand, you can install [`ffmpeg`](https://ffmpeg.org/download.html) , and run the following command:\n",
+    "\n",
+    "```\n",
+    "mkdir video_name\n",
+    "ffmpeg -i video_name.mp4 -vf fps=30 video_name/%6d.png\n",
+    "```\n",
+    "where `-i` denotes input, `-vf fps=30` is the most commonly used frame per second rate.\n",
+    "\n",
+    "\n",
+    "For documentation on preparing the datasets beyond this notebook, please refer to the [documentation on how to prepare datasets](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-prepare-datasets-for-automl-images).\n",
```

Comment:
Is this line applicable to mot tasks ? We don't have any references to video formats in that doc.